src/Simplex/Chat/Library/Commands.hs

```diff
@@ -3251,18 +3260,23 @@ processChatCommand' vr = \case
       msgInfo <- withFastStore' (`getLastRcvMsgInfo` connId)
       CRQueueInfo user msgInfo <$> withAgent (`getConnectionQueueInfo` acId)
 
-protocolServers :: UserProtocol p => SProtocolType p -> ([Maybe ServerOperator], [UserServer 'PSMP], [UserServer 'PXFTP]) -> ([Maybe ServerOperator], [UserServer 'PSMP], [UserServer 'PXFTP])
-protocolServers p (operators, smpServers, xftpServers) = case p of
-  SPSMP -> (operators, smpServers, [])
-  SPXFTP -> (operators, [], xftpServers)
+-- TODO [superpeers] used for CLI specific APIs (also updatedServers below)
+-- TODO              - rework/add similar APIs for superpeers?
+-- TODO              - new "ServerType" type to replace ProtocolType that would unify servers and superpeers?
```

Comment:
Probably unnecessary to add another level of abstraction on chat level, and superpeers can be specially treated where necessary. Adding new protocol type is definitely wrong as it's defined in agent, and superpeers are chat level abstraction

Reply:
yes, better to keep them separate, not as servers